packages/ebayui-core-react/src/ebay-flag/types.ts

```diff
@@ -0,0 +1,270 @@
+/* This file is autogenerated, do not edit - it will be overwritten anyway */
```

Comment:
Is this being generated? I remember we had a script to generate ours.
That said if we move these in a common place then it can be fine

Reply:
It is auto generated yes. We have a script (updated in this PR) to generate it